packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -754,7 +754,7 @@
 
         @include mobile-or-tablet-screen {
             overflow-y: scroll;
-            padding: 2.4rem 0 1.6rem;
```

Comment:
we don't need 1.6 rem?

Reply:
not really we only need 2.4 rem  in all directions